types/plaid-link/index.d.ts

```diff
@@ -1,9 +1,10 @@
 // Type definitions for non-npm package plaid-link-browser 2.0
```

Comment:
This package is now at v3, it seems; are these types updated for that?

Reply:
The [link project](https://github.com/plaid/link) was deprecated nearly [four years ago](https://github.com/plaid/deprecated-link/commit/4286d74a8aeaa1e2f75ab399e7cc09121f279d57) and this github project reference has been incorrect nearly the entire lifetime of these type definitions (also almost four years). While there are several suggested replacements: an iOS SDK, an Android SDK, and a Link examples repo, none seem to fit.

I chose [react-plaid-link](https://github.com/plaid/react-plaid-link) because it was the closest analogue, but it has its own [custom types](https://github.com/plaid/react-plaid-link/blob/master/src/types/index.ts), but in reality, these are the types that serve the [Plaid Link javascript code](https://cdn.plaid.com/link/v2/stable/link-initialize.js) and that is documented by Plaid [here](https://plaid.com/docs/link/web/).

These types benefit those who use the non-React library, but there's no github repo for that project, just the CDN location above. What's the best way to proceed?